packages/angular_devkit/build_angular/src/angular-cli-files/utilities/stats.ts

```diff
@@ -124,10 +161,31 @@ export function statsErrorsToString(json: any, statsConfig: any): string {
     );
   }
 
-  return rs('\n' + errors
-    .map((error: any) => r(`ERROR in ${error}`))
-    .join('\n\n')
-  );
+  let output = '';
+  for (const error of errors as (string | WebpackDiagnostic)[]) {
+    if (typeof error === 'string') {
+      output += r(`ERROR in ${error}\n\n`);
+    } else {
+      const file = error.file || error.moduleName;
+      if (file) {
+        output += c(file);
+        if (error.loc) {
+          output += ':' + yb(error.loc);
+        }
+        output += ' - ';
+      }
+      if (!/^error/i.test(error.message)) {
+        output += r('error: ');
```

Comment:
Nit: maybe start with a capital letter?
```suggestion
        output += r('Error: ');
```